plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonq/FeatureDevSessionContext.kt

```diff
@@ -46,8 +46,6 @@ class FeatureDevSessionContext(val project: Project, val maxProjectSizeBytes: Lo
         "\\.svn",
         "\\.hg/?",
         "\\.rvm",
-        "\\.git/?",
```

Comment:
why do we need to ship their .git objects?

Reply:
will revert, this is definitely not needed